src/dxgi/dxgi_swapchain.cpp

```diff
@@ -425,8 +425,17 @@ namespace dxvk {
 
     Com<IDXGIOutput1> target;
 
-    if (pTarget)
-      pTarget->QueryInterface(IID_PPV_ARGS(&target));
+    if (pTarget) {
+      DXGI_OUTPUT_DESC desc;
+
+      if (FAILED(pTarget->QueryInterface(IID_PPV_ARGS(&target))))
```

Comment:
When is this ever expected to fail? We should probably just error out if the app passes an invalid pointer here.

Reply:
No idea, I was just being defensive.